ui/src/rest/repositories/create-graph-repo.ts

```diff
@@ -0,0 +1,20 @@
+import { RestClient, IApiResponse } from '../rest-client';
+import { ICreateGraphRequestBody } from '../http-message-interfaces/request-interfaces';
+import { Schema } from '../../domain/schema';
+
+export class CreateGraphRepo {
```

Comment:
I thought it would be better to use single responsibility to handle each individual request and responses. 
Also the repository pattern helps create an abstraction layer between the API and the UI's contexts for ease of change for each request and response - if it changes in the API in future.
It also helps unit testing by mocking this persistance layer and it's individual repos for each request/response.